demo/src/main/java/com/orange/ods/demo/ui/components/Component.kt

```diff
@@ -167,7 +167,8 @@ sealed class Variant(
 
     object AppBarsTopRegular : Variant(R.string.component_app_bars_top_regular, "OdsTopAppBar")
 
-    object ButtonsContained : Variant(R.string.component_buttons_high_emphasis, "OdsButton with an high emphasis style")
+    object ButtonsPrimary : Variant(R.string.component_buttons_highest_emphasis, "OdsButton with OdsButtonStyle.Primary")
+    object ButtonsContained : Variant(R.string.component_buttons_high_emphasis, "OdsButton with OdsButtonStyle.Default")
```

Comment:
Should we rename the variant to `ButtonsDefault` ? If I'm correct, contained buttons is a group containing default, primary and functional buttons.

Same comment for the `ButtonsContained()`  method in `ButtonsContained.kt`.